OpenRA.Mods.Common/Commands/DevCommands.cs

```diff
@@ -23,6 +24,25 @@ public class DevCommandsInfo : TraitInfo<DevCommands> { }
 
 	public class DevCommands : IChatCommand, IWorldLoaded
 	{
+		readonly IDictionary<string, (string, Func<string, World, bool>)> commandHandlers = new Dictionary<string, (string, Func<string, World, bool>)>
```

Comment:
This would then be `Action<string, World>`

Reply:
That does work. Updated.